### ISSUE TITLE
AccessibleNumberSpinner won't change value first arrow press

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#413]
AccessibleNumberSpinner won't change value if another key is held down during first press. But any subsequent presses after that will correctly change the value. @mbarlow12 and I are going to look into this today, because it impacts GFL:B.

**Comment by jessegreenberg** [comment]
Shoot, this should be in sun.
